src/commands/migrate.py

```diff
@@ -288,6 +330,28 @@ async def notify_synchronization_failure(self,
         synchronization_failure.action = SYNCHRONIZATION_FAILURE_ACTION_NOTIFIED
         backend_session.add(synchronization_failure)
 
+    async def notify_verification_failure(self, task):
+        """
+        Notifies verification failure
+
+        Args:
+            task: task
+        """
+        task_name = task.get_name()
+        email_body = f"Data verification failed on {task_name}.\n\n" \
+                     f"This is an automated message, please do not reply to this email"
```

Comment:
Probably good for now, but maybe it would be good idea to include some error description or stack trace to the email in the future.